packages/tools/accessibility/src/HtmlTwin/htmlTwinHostComponent.tsx

```diff
@@ -225,7 +283,11 @@ export class HTMLTwinHostComponent extends React.Component<IHTMLTwinHostComponen
         return result;
     }
 
-    private _isGUI(node: Node) {
+    private _isFullscreenGUI(texture: BaseTexture): texture is AdvancedDynamicTexture {
+        return texture instanceof AdvancedDynamicTexture && !!(texture.layer && !texture.layer.isBackground);

```

Comment:
use `getClassName` instead of instanceof (especially if you iterate through all of the scene's textures array)

Reply:
Done!